### PR TITLE
Package in exe

### PR DESCRIPTION
**Pull request by erikafung20**
Addresses Issue #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,7 @@ These include file processing tools (in the file_processing directory). This cod
 - [ ] Write tests to ensure the functionality of the library
 - [ ] Add support for more file types (.docx, .xlsx, .pptx, etc.)
 - [ ] Add more metrics for comparison between files (ex. jaccard)
+
+## Build exe
+- Install `pyinstaller`
```
Comment (pgaviganHC) [inline README.md:46]: should we just add this to the venv requirements file?

Comment (erikafung20) [reply README.md]: great point, just made the change

File: table_processing/requirements.txt
Comment (pgaviganHC) [inline table_processing/requirements.txt:1]: Why do we need a new requirements file? Can the original be used? Also, are sll of these depencdencies needed?

Comment (erikafung20) [reply table_processing/requirements.txt:1]: we don't need this requirements file, forgot to remove after testing sorry.

Comment (pgaviganHC) [reply table_processing/requirements.txt:1]: OK, let's remove that file then.

**Comment by pgaviganHC** [comment]
Shifting this effort to #60

**Pull request closed without merging**